.github/workflows/certification.yml

```diff
@@ -245,10 +252,65 @@ jobs:
           exit 1
         fi
 
+    - name: Prepare Cert Coverage Info
+      run: |
+       mkdir -p path/to/cov_files
+       SOURCE_PATH_LINEAR=$(echo ${{ env.SOURCE_PATH }} |sed 's#/#\.#g')
+       echo "${{ env.cover_per_only }}" >> path/to/cov_files/$SOURCE_PATH_LINEAR.txt
+
+    - name: Upload Cert Coverage Info
```

Comment:
```suggestion
    - name: Upload certification test coverage artifact
```

Reply:
Changed to refer `Upload Cert coverage Artifact` and `Download Cert Coverage Artifact`  - to keep in sync.